test/functional/ui/float_spec.lua

```diff
@@ -379,7 +379,10 @@ describe('float window', function()
       row = 1,
     })
     eq(1001, api.nvim_win_get_config(winid).win)
+    api.nvim_win_set_config(winid, { title = 'foo' })
+    eq(1001, api.nvim_win_get_config(winid).win)
     command('close')
+    api.nvim_win_set_config(winid, { title = 'bar' })
```

Comment:
This now fails again because the target window was closed. Should it? I guess not.

Reply:
Maybe these `win != -1` checks are just wrong? I don't think those fields are ever set or initialized to -1🤔
https://github.com/neovim/neovim/blob/33b0a004eb20fd5a0013b832414596190c059828/src/nvim/api/win_config.c#L240
https://github.com/neovim/neovim/blob/33b0a004eb20fd5a0013b832414596190c059828/src/nvim/api/win_config.c#L417